### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by Logan1x**

### CODE REVIEW

**Comment by Shah-Arsalan** [comment]
Everything looks good . just use rems instead of px

File: avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+.avatar {
+  width: 150px;
+  margin: auto;
+  background: var(--avatar-bg-color);
+}
+
+.avatar img {
+  width: 150px;
+  height: 150px;
+}
```
Comment (lomesshh) [inline avatar/avatar.css]: use rem instead of px

File: avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+.avatar-2 {
+  padding: 1.5rem;
+  border-radius: 50%;
+}
+
+.avatar-3 {
+  padding: 1.5rem;
+  border: 2px solid blueviolet;
+  border-radius: 0.3rem;
+}
```
Comment (lomesshh) [inline avatar/avatar.css:29]: and just a suggestion try to create different sizes of the avatar like 
avatar-circle-sm, avatar-circle-md, avatar-circle-lg

**Review by lomesshh — COMMENTED** [review]
rest code looks perfect to me !

**Comment by Logan1x** [comment]
Thanks, Lomesh and Arsalan. Merging now :)

**Pull request merged**